fred/fred.py

```diff
@@ -85,30 +85,21 @@ def setup_DB(self):
                 time.sleep(5)
         else:  # this happens if the loop is not broken by a successful connection
             raise ConnectionError("Could not connect to the DB")
+        self.logger.info(f"Connected to the DB. Took {attempt} tries.")
         config.migrate()
-
-    def setup_logger(self):
-        logging.root = logging.Logger("FRED")
-        logging.root.setLevel(logging.DEBUG)
-
-        self.logger = logging.root
-
-        self.logger.info("Successfully set up the logger")
-        self.logger.info(f"Prefix: {self.command_prefix}")
+        self.logger.debug("Applied migration.")
 
     def setup_cogs(self):
-        logging.info("Setting up cogs")
+        self.logger.info("Setting up cogs")
         self.add_cog(Commands(self))
         self.add_cog(webhooklistener.Githook(self))
         self.add_cog(mediaonly.MediaOnly(self))
         self.add_cog(crashes.Crashes(self))
         self.add_cog(dialogflow.DialogFlow(self))
         self.add_cog(welcome.Welcome(self))
         self.add_cog(levelling.Levelling(self))
-        self.MediaOnly = self.get_cog("MediaOnly")
-        self.Crashes = self.get_cog("Crashes")
-        self.DialogFlow = self.get_cog("DialogFlow")
-        logging.info("Successfully set up cogs")
```

Comment:
Ignore the fact that those are removed. I messed up what was included in what commit. They're changed to `@property` in 4e123d22f3804ebab2e762d91d69c5ec3629c3fc